css-positioning/fixed.css

```diff
@@ -0,0 +1,12 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  left: 35%;
```

Comment:
another way you could ensure it's centered is set `left: 0` and `right: 0` because that'll ensure the width takes up the entire width of the body!

Reply:
Ah!